code/game/machinery/shieldgen.dm

```diff
@@ -145,9 +149,10 @@
 
 
 /obj/machinery/shieldgen/proc/shields_up()
-	if(active) return 0 //If it's already turned on, how did this get called?
+	if(active)
+		return 0 //If it's already turned on, how did this get called?

```

Comment:
`return FALSE` or just a return if this isn’t returning any checked values as it appears is the case